docs/index.md

```diff
@@ -4,54 +4,31 @@ page_description: Introduces the Docker Registry
 page_keywords: registry, images, repository
 IGNORES-->
 
-# Docker Registry 2.0
+# Docker Registry
 
-Docker Registry stores and distributes images centrally. It's where you push images to and pull them from; Docker Registry gives team members the ability to share images and deploy them to testing, staging and production environments.
```

Comment:
The team members and testing/staging/production thing feels irrelevant.
So does words like "centrally".